### DISCUSSION TITLE
How to add disclaimer or other text to title slide

### DISCUSSION THREAD

**Discussion by anielsen001** [quarto-dev/quarto-cli#6742] [Q&A]
### Description

Can someone recommend a way to add a block of text to the title slide would contain some boilerplate language like an employer required disclaimer? 

I'd like to have a small print block of text under the title, author, etc that says something like "The views represent the views of the author and do not represent the view of the employer"

**Comment by mcanouil** [comment]
For which format?
There is the footer in `revealjs`:
- https://quarto.org/docs/presentations/revealjs/index.html#footer-logo
An extension:
- https://github.com/quarto-ext/attribution
Any HTML/CSS with absolute position.

**Reply by anielsen001** [reply]
I experimented with the footer, but the text is too long. It's several sentences. I would prefer something like a textblock environment in latex.

**Reply by mcanouil** [reply]
Then do that.
Define your own CSS class and use fenced divs, see <https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans>.
